nemo/lightning/megatron_parallel.py

```diff
@@ -589,7 +589,7 @@ def init_ddp(self):
                     self.ddp_config,
                     module,
                     data_parallel_group=parallel_state.get_data_parallel_group(with_context_parallel=True),
-                    expert_data_parallel_group=parallel_state.get_data_modulo_expert_parallel_group(),
+                    expert_data_parallel_group=parallel_state.get_expert_data_parallel_group(),
```

Comment:
I don't think MCore in CI container needs this change

Reply:
@ShriyaPalsamudram

Reply:
Given that the [deprecation warning is already out](https://github.com/NVIDIA/Megatron-LM/blob/main/megatron/core/parallel_state.py#L1712), isn't it good to change it now before it ends up getting deprecated and then throwing an error?

Reply:
the function doesn't exist in the current MCore commit we have in the CI container. We are using `c1728c12f1f1cdbb786e52f1ffe512295d76bef3`. `get_expert_data_parallel_group()` was introduced in `7f22e210cddc3215adda25d9e16ea512dc32458c`.

Reply:
Oh I see, thanks for pointing this out. Reverted in the latest commit.